source/testnet/see-also/release-notes.rst

```diff
@@ -17,6 +14,38 @@ Release Notes
    :local:
    :backlinks: none
 
+Open Testnet v4
+===============
+
+January 13th, 2020
+
+Smart contracts:
+
+* Smart contracts support on chain
+* Rust supported as off-chain SC language
+* Cargo-concordium tool for building and testing smart contracts off-chain
+* Documentation for smart contracts added to `developer docs <https://concordium.github.io/en/testnet4/smart-contracts/index.html>`_
+* SC transactions added to Concordium client
+
+Tokenomics (to match tokenomics model):
+
+* Rewards for baking and finalization changed
+* Minting changed
+* Chain parameters updated
```

Comment:
Is this trying to say that we have new chain parameters, or that chain parameter updates are possible now?

Regarding the latter, we could change some parameters before as well.

Reply:
Good question. @smh1001, can you elaborate?

Reply:
It is trying to say that we have updated chain parameters. I don't know how to put it more precisely in one sentence.

Reply:
It could maybe say "Extended list of adjustable chain parameters". Is that better?

Reply:
This has been changed to Extended list of adjustable chain parameters. Let me know what you think @abizjak

Reply:
I think that's fine.